sdk/ai/azure-ai-projects/azure/ai/projects/models/_models.py

```diff
@@ -6095,7 +6095,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
 
 class VectorStoreFileError(_model_base.Model):
-    """Details on the error that may have occurred while processing a file for this vector store.
+    """Details on the error that may have ocurred while processing a file for this vector store.
```

Comment:
Based on my dictionary, the correct spelling is `occurred`

Reply:
Fixed.